autowebcompat/utils.py

```diff
@@ -179,12 +179,16 @@ def make_infinite(gen_func, elems):
         yield from gen_func(elems)
 
 
-def read_labels(file_name='labels.csv'):
+def read_labels(encoding, file_name='labels.csv'):
```

Comment:
Let's use 'Y vs D+N' as the default here, and let's replace **encoding** with something like **classification_type**